apps/server/src/app.ts

```diff
@@ -30,6 +31,26 @@ if (corsValue?.[0] === ALLOW_ALL_CORS) {
   corsValue = undefined;
 }
 
+// Mask certain search params in logs
+const maskedSearchParams: Partial<Record<string, Set<string>>> = {
+  '/oauth/spotify/callback': new Set(['code']),
```

Comment:
I haven't seen any other interesting query parameters. If there are any, please mention them so that they can be included here.